add support for .Net Core

manuc66 opened the pull request.

psoholt commented:
Thanks for adding support for .net core. Could you also leave support for .csproj files, they seemed to have been removed.

manuc66 commented:
In fact project.json is the replacement of csproj files. 
For sln compatibility Visual Studio 2015 add .xproj file.
.nuspec is also replaced with project.json.

project.json simplify multitargetting so I also added some other frameworks. Tests are now only done with dnxcore50 and NUnit 3.2.0 (due to NUnit).

psoholt commented:
Ok, looks like something we should support. But we have to wait until it is supported a bit wider. Xamarin alpha version doesn't support it and the new Rider EAP don't support it yet. As soon as they are updated we can support the .net core.

willl commented:
Any update to this? `.xproj` and `project.json` are both now gone and `.csproj` has come back again.
